Ryujinx.HLE/HOS/Horizon.cs

```diff
@@ -493,24 +515,24 @@ Nacp ReadControlData()
 
         private void LoadExeFs(IFileSystem codeFs, out Npdm metaData)
         {
-            if (codeFs.FileExists("/main.npdm"))
-            {
-                Logger.PrintInfo(LogClass.Loader, "Loading main.npdm...");
+            Result rc = codeFs.OpenFile(out IFile npdmFile, "/main.npdm", OpenMode.Read);
```

Comment:
```suggestion
            Result resultCode = codeFs.OpenFile(out IFile npdmFile, "/main.npdm", OpenMode.Read);
```
and add an empty line after.